easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -270,8 +331,52 @@ def configure_step(self):
             raise EasyBuildError("Failed to move non-espresso directories: %s", err)
 
     def install_step(self):
-        """Skip install step, since we're building in the install directory."""
-        pass
+        """Custom install step for Quantum ESPRESSO."""
+
+        # Copy all binaries
+        bindir = os.path.join(self.installdir, 'bin')
+        copy_dir(os.path.join(self.cfg['start_dir'], 'bin'), bindir)
+
+        # Pick up files not installed in bin
+        upftools = []
+        if 'upf' in self.cfg['buildopts'] or 'all' in self.cfg['buildopts']:
+            upftools = ["casino2upf.x", "cpmd2upf.x", "fhi2upf.x", "fpmd2upf.x", "ncpp2upf.x",
+                        "oldcp2upf.x", "read_upf_tofile.x", "rrkj2upf.x", "uspp2upf.x", "vdb2upf.x",
+                        "virtual.x"]
+            if LooseVersion(self.version) > LooseVersion("5"):
+                upftools.extend(["interpolate.x", "upf2casino.x"])
+            if LooseVersion(self.version) >= LooseVersion("6.3"):
+                upftools.extend(["fix_upf.x"])
+        upf_bins = [os.path.join('upftools', x) for x in upftools]
+        for upf_bin in upf_bins:
+            copy_file(os.path.join(self.cfg['start_dir'], upf_bin), bindir)
+
+        wanttools = []
+        if 'want' in self.cfg['buildopts']:
+            wanttools = ["blc2wan.x", "conductor.x", "current.x", "disentangle.x",
+                         "dos.x", "gcube2plt.x", "kgrid.x", "midpoint.x", "plot.x", "sumpdos",
+                         "wannier.x", "wfk2etsf.x"]
+            if LooseVersion(self.version) > LooseVersion("5"):
+                wanttools.extend(["cmplx_bands.x", "decay.x", "sax2qexml.x", "sum_sgm.x"])
+        want_bins = [os.path.join('WANT', 'bin', x) for x in wanttools]
+        for want_bin in want_bins:
+            copy_file(os.path.join(self.cfg['start_dir'], want_bin), bindir)
+
+        w90tools = []
+        if 'w90' in self.cfg['buildopts']:
+            if LooseVersion(self.version) >= LooseVersion("5.4"):
+                w90tools = ["postw90.x"]
+                if LooseVersion(self.version) < LooseVersion("6.1"):
+                    w90tools = ["w90chk2chk.x"]
+        w90_bins = [os.path.join('W90', x) for x in w90tools]
+        for w90_bin in w90_bins:
+            copy_file(os.path.join(self.cfg['start_dir'], w90_bin), bindir)
+
+        yambo_bins = []
+        if 'yambo' in self.cfg['buildopts']:
+            yambo_bins = ["a2y", "p2y", "yambo", "ypp"]
```

Comment:
@akesandgren These binaries are in `YAMBO/bin`, so you'll need this above this `for` loop:

```python
yambo_bins = [os.path.join('YAMBO', 'bin', x) for x in yambo_bins]
```